Orange/widgets/visualize/owdistributions.py

```diff
@@ -114,11 +115,10 @@ def __init__(self):
 
         self.distributions = None
         self.contingencies = None
-        self.var = self.cvar = None
         varbox = gui.vBox(self.controlArea, "Variable")
 
-        self.varmodel = itemmodels.VariableListModel()
-        self.groupvarmodel = []
+        self.varmodel = DomainModel(separators=False, valid_types=DomainModel.PRIMITIVE)
```

Comment:
This is one case where we don't want the class to be at the beginning; we use class for grouping (by default).